packages/create-next-app/templates/app-tw/ts/tailwind.config.ts

```diff
@@ -1,5 +1,6 @@
-/** @type {import('tailwindcss').Config} */
-module.exports = {
+import type { Config } from 'tailwindcss'
+
+const config: Config = {
```

Comment:
The "conventional way" (i.e how they show it in their docs) is to do `export default {} satisfies Config` but since your monorepo is on TS4.8 i couldnt use satisfies without getting errors